rsmtool/notebooks/feature_descriptives.ipynb

```diff
@@ -84,7 +84,13 @@
     "    legend.legendHandles[0].set_color(colors[0])\n",
     "    legend.legendHandles[1].set_color(colors[1])\n",
     "\n",
-    "    plt.tight_layout(h_pad=1.0)\n",
+    "    # we want to try to force `tight_layout()`, but if this fails\n",
+    "    # we don't want the entire notebook to fail\n",
+    "    try:\n",
+    "        plt.tight_layout(h_pad=1.0)\n",
+    "    except:\n",
```

Comment:
Can we specify the actual exception name here if we can? This kind of non-specific exception catching is generally frowned upon.